.tekton/tasks/verify-smoke-e2e.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   params:
   - name: PLATFORM
     description: which environment the test is running on.
-    default: "stage"
+    default: "verify-stage"
```

Comment:
Shouldn't it be "staging"?

Reply:
I am just updating this as per https://github.com/redhat-appstudio/e2e-tests/pull/944/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR294

Reply:
OK, it looks like a naming inconsistency.